{
  "summary": {
    "outcome": "Failed",
    "testsRan": 1,
    "passing": 0,
    "failing": 1,
    "skipped": 0,
    "passRate": "0%",
    "failRate": "100%",
    "skipRate": "0%",
    "testStartTime": "Fri Sep 17 2021 11:28:28 a.m.",
    "testExecutionTimeInMs": 95,
    "testTotalTimeInMs": 95,
    "commandTimeInMs": 211,
    "hostname": "https://d5e000001ndlkeak-dev-ed.my.salesforce.com",
    "orgId": "00D5e000001NdLkEAK",
    "username": "madan.sreya17@gmail.com",
    "testRunId": "7075e00001wlFHR",
    "userId": "0055e00000360bNAAQ"
  },
  "tests": [
    {
      "id": "07M5e00000FGNd0EAH",
      "queueItemId": "7095e000000AujpAAC",
      "stackTrace": "Class.TrainingDealTest.TestFunction: line 12, column 1",
      "message": "System.DmlException: Insert failed. First exception on row 0; first error: FIELD_INTEGRITY_EXCEPTION, Course: id value of incorrect type: a035e000001vs2gAAA: [Course__c]",
      "asyncApexJobId": "7075e00001wlFHRAA2",
      "methodName": "TestFunction",
      "outcome": "Fail",
      "apexLogId": "07L5e000004QcbfEAC",
      "apexClass": {
        "id": "01p5e00000ODZI3AAP",
        "name": "TrainingDealTest",
        "namespacePrefix": null,
        "fullName": "TrainingDealTest"
      },
      "runTime": 95,
      "testTimestamp": "2021-09-17T15:28:28.000+0000",
      "fullName": "TrainingDealTest.TestFunction",
      "diagnostic": {
        "exceptionMessage": "System.DmlException: Insert failed. First exception on row 0; first error: FIELD_INTEGRITY_EXCEPTION, Course: id value of incorrect type: a035e000001vs2gAAA: [Course__c]",
        "exceptionStackTrace": "Class.TrainingDealTest.TestFunction: line 12, column 1",
        "className": "TrainingDealTest",
        "compileProblem": "",
        "lineNumber": 12,
        "columnNumber": 1
      }
    }
  ]
}